core/templates/calendar.html

```diff
@@ -1,12 +1,14 @@
 <link rel="stylesheet" type="text/css" href="/css/reset.css">
 <link rel="stylesheet" type="text/css" href="/css/calendar.css">
+
 <script>
-  <% var roomsHLRC =  ["Recording Studio", "1141A", "1141B", "1141C", "1141D"] %> <!--Should these be passed in from app.js?-->
+  <% var roomsHLRC =  ["Recording Studio", "1141A", "1141B", "1141C", "1141D"] %> //Should these be passed in from app.js?-->
   <% var roomsHTRSC = ["B160", "B164", "B181", "B013", "B161", "1161", "B003", "1131A", "1133A", "1133B"] %>
   window.rooms = {};
   window.rooms.HLRC = ("<%- (roomsHLRC.toString()) %>").split(",");
   window.rooms.HTRSC = ("<%- roomsHTRSC.toString() %>").split(",");
 </script>
+<script src="jquery-1.11.2.min.js">//VERY HACKY, but it will have to do for now. Some thrid party is preventing JQuery from loading all the way. Now Jquery is "double added" this is in not good practice. #HammerTime</script>
```

Comment:
I haven't tested it to verify, though. You might want to rebase just in case there's something else messing it up.